npm/pod_test.go

```diff
@@ -202,3 +202,183 @@ func TestDeletePod(t *testing.T) {
 	}
 	npMgr.Unlock()
 }
+
+func TestAddHostNetworkPod(t *testing.T) {
+	npMgr := &NetworkPolicyManager{
+		nsMap:            make(map[string]*namespace),
+		podMap:           make(map[string]string),
+		TelemetryEnabled: false,
+	}
+
+	allNs, err := newNs(util.KubeAllNamespacesFlag)
+	if err != nil {
+		panic(err.Error)
+	}
+	npMgr.nsMap[util.KubeAllNamespacesFlag] = allNs
+
+	ipsMgr := ipsm.NewIpsetManager()
+	if err := ipsMgr.Save(util.IpsetTestConfigFile); err != nil {
+		t.Errorf("TestAddHostNetworkPod failed @ ipsMgr.Save")
```

Comment:
add error to test logging. gives more context as to why something failed. same for below

Reply:
Logged a work item to fix this for all NPM tests.